packages/components/src/sandbox/README.md

```diff
@@ -1,4 +1,4 @@
-# Sandbox
+# SandBox
```

Comment:
The public export is capitalized as "SandBox", so I've fixed up the capitalization inconsistencies.